components/Nav/NavMobile/NavMobile.js

```diff
@@ -49,19 +54,40 @@ export default class NavMobile extends Component {
             <ul className={styles.ul}>
               <li className={styles.li}>
                 <Link href="/">
-                  <a className={styles.a}>Home</a>
+                  <button
```

Comment:
I'd prefer that these remain anchor tags. It's semantically correct.

Next.js treats the URL as the master of state. Knowing that, I would recommend converting Nav to a class component, creating a callback to change the mobile nav's state, and then leverage Next's router within `Navigation` to listen for a route change and fire that call back. Here's how we do it at my work:

**Navigation.js**
```javascript
  // ...

  componentDidMount() {
    Router.events.on('routeChangeComplete', this.closeMobileMenu);
  }

  componentWillUnmount() {
    Router.events.off('routeChangeComplete', this.closeMobileMenu);
  }
 
  // ...
```

Reply:
I forget why I changed the anchor tags to button, it may have been some misguided notion about Aria.

I will see what I can do, thanks for the detailed explanation.